added all board files and a software structure

Eemac opened the pull request:
(NEEDS UPDATES ON THE SOFTWARE SIDE) - let me know what y'all think.

Rixhee committed:
Updated temp firmware for mkvii

Rixhee committed:
Updated main BUILD

Rixhee committed:
Updated canviewer for muxes

Rixhee committed:
Fix rules_python version

Rixhee committed:
Commenting charging msg

Rixhee committed:
Added mkvii dbc and 64m1 dependency

Rixhee committed:
Fixed canviewer for temps and tiny bms code to send can message for temp readings

Rixhee committed:
Fixed bms.c to read temps

Configuring all muxes at the start and removed mux iteration

Rixhee committed:
Fixed canviewer formatting

Eemac committed:
added all board files and a software structure (NEEDS UPDATES ON THE SOFTWARE SIDE)

H-TejadaDeras commented:
Hi Ian,
I like the overall structure of the repo! Just a few things regarding the boards I worked with (and some other things I found).
- When I was modifying the BSPD Schematics, I deleted a 1-2 pages (connector and shtdwn_lsd kicad files). I didn't remove them because I didn't want to break anything, feel free to remove them if they don't break anything!
- When making the LV Motherboard PCB Design, I used a footprint to create the PCB key silkscreen, consider adding that, it is in the Footprints Folder 
- throttle board has ibom included
- Do we want to leave the "sym-lib-table" files (one is found in the Sus. Strain Board and Power Thermistor Board directories)
- I think we should keep the file structure between the hardware and software folders the same (as in we should have the subteam names and then the folders for the software for each individual board just like in the hardware folder)

Commit:
added software directory for MKVII

H-TejadaDeras committed:
Dashboard is working!

Eemac committed:
new motor controller syntax

H-TejadaDeras committed:
Add updated changes with canviewer (to work on mkvii)

Eemac committed:
flipped polarity on the dashboard input sensing pins

H-TejadaDeras committed:
Add Modifications to Dashboard and Throttle Firmware
Dashboard: Added RTD Button LED Functionality
Throttle: Changes from Althea I refactoring; other bug fixes

aeveical committed:
Debugged throttle

aeveical committed:
Merge branch 'ian/mkvii_structure' of github.com:olin-electric-motorsports/olin-electric-motorsports into ian/mkvii_structure

aeveical committed:
Debugged Throttle

H-TejadaDeras committed:
Fix pcint#_callback function issues that caused Shutdown Sense can message irregularities

H-TejadaDeras committed:
Merge remote-tracking branch 'refs/remotes/origin/ian/mkvii_structure' into ian/mkvii_structure

aeveical committed:
Initial implementation of heartbeat and error LED

H-TejadaDeras committed:
Made brake_implausibility_occurred not a static var

Commit:
lots of progress on the airctrl, sus_strain done we think!

Rixhee committed:
uncommented set faults

Rixhee committed:
Updated bms main to constantly send charging command

Rixhee committed:
Attempt to control fan using pwm

Rixhee committed:
Uncommmented btldr code

Rixhee committed:
initial attempt to make btldr 64m1 compat

Commit:
BMS cor and air control

Rixhee committed:
Artificially fixed first and last V

Rixhee committed:
added __vectors in 64m1 ld

Rixhee committed:
Fixed OT fault

H-TejadaDeras committed:
Calibrated Accelerator Pedal Potentiometers

Rixhee committed:
Fixed avrdude config for 64m1 btld compat

Rixhee committed:
btld build dynamic for 64m1 and 16m1

Rixhee committed:
Added btldr compat for both 16 and 64m1

Rixhee committed:
Merge branch 'rishit/bootloader' into rishit/final-bms

Rixhee committed:
Uncommented can voltage msg

Commit:
air control final code

Commit:
updated btldr to working version

Commit:
Merge branch 'rishit/final-bms' into ian/mkvii_structure

Commit:
mkvii bms final?

Commit:
bms merging strangeness

Commit:
working bms and air ctrl images with btldr!

Rixhee committed:
Flipped fan logic and it works!

H-TejadaDeras committed:
Commented out Heartbeat LED Code

H-TejadaDeras committed:
Merge branch 'ian/mkvii_structure' of github.com:olin-electric-motorsports/olin-electric-motorsports into ian/mkvii_structure

H-TejadaDeras committed:
Added Untested Wheel Speed Firmware

Jake30123 committed:
Throttle ADC fault latching fix

Jake30123 committed:
Motor direction and avr version

H-TejadaDeras committed:
Renamed Brakes Folder to BSPD; Minor changes to BSPD Firmware Comments

H-TejadaDeras committed:
Added Jacob Keller's Latest PDU Firmware Version

H-TejadaDeras committed:
Added Updated Dependecies for PDU Firmware; Reverted Requested AVR Package to 5.4.0

H-TejadaDeras committed:
Modified Final Version of PDU Firmware to Build Successfully using Bazel

H-TejadaDeras committed:
Reverted PDU Firmware to suspected firmware version on Mk. VII at Comp

jalikins committed:
workflow updates

H-TejadaDeras committed:
Moved Previous Versions of Parts Libraries to .old Folders

H-TejadaDeras committed:
Added OEM v2 KiCad Parts Libraries

H-TejadaDeras committed:
Added KiCad OEM Project Template

H-TejadaDeras committed:
Added OEM v2 KiCad Symbols Library

H-TejadaDeras committed:
Moved Symbols Library

H-TejadaDeras committed:
Updated OEM v2 Footprints Library to correct issues with 3D Models

H-TejadaDeras committed:
Added PCIe and Ampseal Connectors to KiCad Parts Libraries

H-TejadaDeras committed:
Updated STM32 and PCIe x1 Symbols

H-TejadaDeras committed:
Added Precharge-Discharge Relays to Parts Library

H-TejadaDeras committed:
Added Various Connectors and Updated Resistors

H-TejadaDeras committed:
Added STM32G441CBT6 Symbol

H-TejadaDeras committed:
Added New BMS Connector

H-TejadaDeras committed:
Added Molex Connectors

H-TejadaDeras committed:
AddedMissing Zener Diode

H-TejadaDeras committed:
Corrected issue with inductor symbol

Jackk36 committed:
adding two wire screw terminal kiCAD schematic and PCB layout to OEM library

H-TejadaDeras committed:
Added E-Meter Temperature Sensor to Parts Library.

H-TejadaDeras committed:
Resyncronized Parts Library

aditilagisetty committed:
commiting baseline pdu code

aditilagisetty committed:
Merge branch 'ian/mkvii_structure' of github.com:olin-electric-motorsports/olin-electric-motorsports into ian/mkvii_structure

H-TejadaDeras committed:
Updated KiCad Library to v2.1
